### ISSUE TITLE
Commands to move read receipt and/or fully read marker to point to a particular event

### ISSUE THREAD

**Issue by erkinalp** [vector-im/element-web#21540]
#### What would you like to do?
`/readreceipt <event-id>`
`/fullyread <event-id>`
When used without arguments, a pop-up with the current marker positions should be returned.
#### How would you like to achieve it?
`/readreceipt` should also check forward-only invariant client-side, before sending the request to the server side.

### Have you considered any alternatives?

"Mark as unread" but that would be coarse.

**Comment by robintown** [comment]
I assume the use case for this is to move read receipts backwards? That action is actually prevented at the Synapse level, in order to mitigate races between clients.

**Comment by erkinalp** [comment]
No, this is to move the fully read markers backwards, which **is** allowed per spec. No spec issues really.
